widget/info.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "recordSummaryTile",
     "title": "Record Summary Tile",
-    "subTitle": "Widget lists, number of customers and Alerts",
+    "subTitle": "The Record Summary widget extracts JSON data from your FortiSOAR module records and presents it in a convenient tile format.",
     "version": "1.0.0",
     "published_date": 1692194643,
     "metadata": {
         "help_online": "https://github.com/fortinet-fortisoar/widget-record-summary-tile/blob/release/1.0.0/README.md",
-        "description": "This widget displays data in a card-tile, if data is provided in the required format.",
+        "description": "The Record Summary widget extracts JSON data from your FortiSOAR™ module records and presents it in a convenient tile format. Complex information is distilled into a glanceable summary, making it easy to understand and act upon.",
```

Comment:
This widget renders JSON record from FortiSOAR™ module and presents it in a convenient tile format. Complex information is distilled into a glanceable summary, making it easy to understand and act upon.

Please get it reviewed by infodev

Reply:
Reviewed and changed